flatnav/util/SquaredL2SimdExtensions.h

```diff
@@ -45,11 +77,23 @@ static float computeL2_Avx2(const void *x, const void *y,
     v2.loadu(pointer_y);
     difference = v1 - v2;
     sum += difference * difference;
+
```

Comment:
Loop unrolling so that at every iteration we process more data. This is supposed to be slightly faster. In fact, there is an entire gcc flag for unrolling loops.